### PR TITLE
Popups With Details

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature I did the following

- Made sure to remove linter errors
- Add comment button
- Add modal with popup feature
- Added image in popup with close button

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,6 +10,32 @@ const pokemonImage = (pokeID, container) => {
...
+          <button class="closePopupBtn">Close</button>
+      </div>
+  `;
+  const popup = document.querySelector('.popup');
+  const pokeType = document.createElement('ul');
+  pokeType.innerHTML = 'Type(s)';
+  pokeType.classList.add('types');
+  popup.appendChild(pokeType);
+
+  getCardTypes(pokeData.types, pokeType);
```
Comment (ismayito) [inline src/index.js]: Is there a way we can modularize the popup function for best javascript practices

File: src/index.js [hunk 2]
```diff
@@ -11,6 +10,32 @@ const pokemonImage = (pokeID, container) => {
...
+
+  const closePopupBtn = modalBox.querySelector('.closePopupBtn');
+  closePopupBtn.addEventListener('click', () => {
+    modalBox.style.display = 'none';
+  });
+
+  modalBox.style.display = 'flex';
+};
+
```
Comment (ismayito) [inline src/index.js:52]: Likewise is to this function. if possible create a module of its own

File: src/style.css [hunk 3]
```diff
@@ -109,3 +111,22 @@ body {
...
+  display: none;
+  position: fixed;
+  top: 0%;
+  left: 0;
+  margin: 0 auto;
+  z-index: 3;
+  background-color: hsla(173, 56%, 6%, 0.63);
+  width: 100%;
+  height: 100%;
+}
```
Comment (ismayito) [inline src/style.css:128]: Nice styling🧡

**Review by ismayito — COMMENTED** [review]
CHANGES REQUIRED: 
Thank you so much for the feature added good job 💪💪
However there are some changes required.

File: src/index.js (hunk 1 above)
Comment (Chimoneg27) [reply src/index.js]: I'll see. If not We'll keep it as is

**Review by ismayito — APPROVED** [review]
There is nothing left to say about this project apart from saying that you can merge it and move forward.

**Pull request merged**